prime-router/src/main/kotlin/Report.kt

```diff
@@ -39,6 +40,20 @@ const val REPORT_MAX_ITEMS = 10000
 const val REPORT_MAX_ITEM_COLUMNS = 2000
 const val REPORT_MAX_ERRORS = 100
 
+// constants used for parsing and processing a report message
+const val ROUTE_TO_SEPARATOR = ","
+const val DEFAULT_SEPARATOR = ":"
+
+// options are used to process and route the report
+enum class Options {
```

Comment:
Add kdoc maybe even explain where options come from or how they are used

Reply:
It almost feels this enum should not be in this file as it is not used or contained here.  Maybe a more suitable file?

Reply:
Added comment explaining why it is here, and referencing #2926